### PR TITLE
User stories 45-48

### PR DESCRIPTION
**Pull request by tylertomlinson**
## All Submissions:

* [x] Merge the target branch into the PR branch. Fix any conflicts that might appear.
* [x] Have you checked to ensure that other pull request are approved for merge [Pull Requests](../../../pulls)?
* [x] What stories does the PR include

### New Feature Submissions:

* [x] Have you added an explanation of what your changes do?
* [x] Have you lint your code locally prior to submission?

### Testing:

* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?
* [x] All new and existing tests passed.

**Test coverage** = 100% features 100% models
--------------------------------------------------------------------------------
### Change Log

Merchant can create, edit and delete items 
Default picture placed on item table 
Testing all above

### CODE REVIEW

File: app/controllers/merchant/items_controller.rb [hunk 1]
```diff
@@ -1,25 +1,64 @@
...
     item = Item.find(params[:id])
     item.destroy
-    flash[:success] = "Item Successfully Deleted."
+    flash[:success] = "#{item.name} Successfully Deleted."
     redirect_to "/merchants/#{item.merchant.id}/items"
   end
 
+  private
+
+  def toggle_status
```
Comment (danielpselph) [inline app/controllers/merchant/items_controller.rb:51]: I like the use of this helper method

File: config/routes.rb [hunk 2]
```diff
@@ -27,7 +27,8 @@
 
   namespace :merchant do
     get "/", to: "dashboard#index"
-    resources :items, only: [:index, :show, :update, :destroy]
+    resources :items, only: [:index, :new, :create, :show, :edit, :update, :destroy]
```
Comment (alex-latham) [inline config/routes.rb]: We can drop the `only: ...` part since we're using all 7 restful routes now

**Review by Mariana-21 — APPROVED** [review]
Nice tests and clean code

File: spec/features/merchant/items_new_spec.rb [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      fill_in 'Inventory', with: @item.inventory
+
+      click_on 'Create Item'
+
+      expect(current_path).to eq("/merchant/items")
+
+      expect(page).to have_content("#{@item.name} has been created.")
+    end
+  end
+end
```
Comment (alex-latham) [inline spec/features/merchant/items_new_spec.rb:91]: Good tests

**Review by alex-latham — APPROVED** [review]
Good to go. Thank you.

**Pull request merged**